src/python/raicontours/BUILD

```diff
@@ -15,5 +16,11 @@ python_distribution(
 
 resources(
```

Comment:
why is this being done here instead of as python_sources() at the top.
That seems to be the way it was done elsewhere, as well as for ::lib in this BUILD

Reply:
No reason in particular. I was copying the style of one of the pants maintainers:

https://github.com/sureshjoshi/pants-example-plugin/blob/main/helloworld/BUILD

Reply:
Actually, that wasn't the one.

This is the one:

https://github.com/pantsbuild/pants/blob/main/src/python/pants/BUILD